src/Resources/config/actions.xml

```diff
@@ -20,5 +20,27 @@
                 <argument type="service" id="service_container"/>
             </call>
         </service>
+        <service id="sonata.admin.action.append_form_field_element" class="Sonata\AdminBundle\Action\AppendFormFieldElementAction" public="true">
```

Comment:
Is everyone ok that my idea of using FQCNs as ids was bad, and we are no longer doing that? See https://github.com/symfony/symfony-docs/pull/9886